pkg/ddevapp/settings_test.go

```diff
@@ -8,19 +8,30 @@ import (
 
 	"io/ioutil"
 
+	"fmt"
+
 	. "github.com/drud/ddev/pkg/ddevapp"
 	"github.com/drud/ddev/pkg/fileutil"
 	"github.com/drud/ddev/pkg/testcommon"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/stretchr/testify/assert"
 )
 
+var appTypeSettingsLocations = map[string][]string{
+	"drupal6":  {"sites/default/settings.php", "sites/default/settings.ddev.php"},
+	"drupal7":  {"sites/default/settings.php", "sites/default/settings.ddev.php"},
+	"drupal8":  {"sites/default/settings.php", "sites/default/settings.ddev.php"},
+	"backdrop": {"settings.php", "settings.ddev.php"},
+}
```

Comment:
No need to change this, but it would be a lot more flexible if the members were named as opposed to a positional array. But we can do that when we come back and need to change it. Positional arrays are pretty lousy for future expansion, using numeric indexes that are magical and hard to maintain.